ozaria/site/common/ozariaUtils.js

```diff
@@ -0,0 +1,124 @@
+/**
```

Comment:
Refactored to ozaria folder, and changed the export value (from a single object to multiple individual exports). Code review not necessary.